src/components/Tabs/Tabs.less

```diff
@@ -31,30 +31,74 @@
 		}
 
 		&:hover {
-			background-color: @color-gray-5;
+			background-color: @color-lightGray;
+			.Tabs-mixin-tab-arrow-background(@color-lightGray);
 		}
 
 		&:active {
-			box-shadow: @shadow-inset;
+			background-image: linear-gradient(#dbdbdb 0, @color-lightGray @Tabs-spacer-height);
```

Comment:
this was a little bit of hack.  when I change the `box-shodow` firefox did not like itl  So changed to a background gradient.  I pulled the colors off the JPG.  Since they were on the gray backgorund they were a little different than our standard grays.